static/bidder-params/rise.json

```diff
@@ -4,10 +4,13 @@
   "description": "A schema which validates params accepted by the Rise adapter",
   "type": "object",
   "properties": {
-    "publisher_id": {
+    "org": {
       "type": "string",
-      "description": "The publisher ID"
+      "description": "The organization ID"
     }
   },
-  "required": ["publisher_id"]
+  "oneOf": [
+    { "required": ["publisher_id"] },
+    { "required": ["org"] }
+  ]
```

Comment:
The `publisher_id` is  mentioned as `oneOf` required. So should be mentioned  under `properties` with deprecated prefix in description. Refer following example.

https://github.com/prebid/prebid-server/blob/225e31e9afea25f8b844adcd3d274e8db3f7135e/static/bidder-params/appnexus.json#L12-L15

Reply:
@onkarvhanumante Sure, done